src/pages/components/select/style.mdx

```diff
@@ -113,16 +111,23 @@ and any proper nouns capitalized. Select text should be three words or less.
 
 | Class                | Property                    | px / rem | Spacing token |
 | -------------------- | --------------------------- | -------- | ------------- |
-| `.bx--select-input`  | height                      | 32 / 2   | –             |
 | `.bx--select-input`  | padding-left                | 8 / 0.5  | `$spacing-03` |
 | `.bx--select__arrow` | padding-left, padding-right | 8 / 0.5  | `$spacing-03` |
 
 <div className="image--fixed">
 
+  
 ![Structure and spacing measurements for Inline Select](images/select-style-6.png)
+  
 
 </div>
 
 <Caption>
   Structure and spacing measurements for inline select (focused) | px / rem
 </Caption>
+
+## Size
+| Element              | Size                        | Height (px / rem)  |
+| -------------------- | --------------------------- | ------------------ | 
+| Input                | Small (sm)                  | 32 / 2             | 
+|                      | Medium (md)                 | 40 / 2.5           |
```

Comment:
I believe we also have an `lg` / `48px` [variant](https://react.carbondesignsystem.com/?path=/story/components-select--default) as well 

<img width="735" alt="Screen Shot 2021-06-04 at 10 04 12 AM" src="https://user-images.githubusercontent.com/11928039/120838030-38585000-c51c-11eb-9e3b-930b8317efb4.png">